packages/atomic/src/components/search/atomic-search-box-recent-queries/atomic-search-box-recent-queries.ts

```diff
@@ -78,17 +89,23 @@ export class AtomicSearchBoxRecentQueries {
     return this.icon || Clock;
   }
 
-  private initialize(): SearchBoxSuggestions {
+  private initializeRecentQueries(): SearchBoxSuggestions {
```

Comment:
Why did you need to rename it?

I suspect this might be because you're using the bindings() decorator.